.ci/build_docker.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "$(dirname "${0}")"
+
+cp /usr/local/bin/runbld ./
+cp /usr/local/bin/bash_standard_lib.sh ./
+
+docker build -t kibana-ci -f ./Dockerfile .
```

Comment:
I assume this won't do any rate-limited operations if the docker container is already cached within the image but will build if there are local changes?

Reply:
Right, the base node image should be cached on the worker, so it shouldn't need to pull anything